recipes/mechwolf/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "mechwolf" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "96641d16c38ec8a5ce2cd3e9130bc79f4d0837cae14608555d8d0b597db1eac5"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py<37]
```

Comment:
This can be `noarch: python` if you restrict python below with `python >=3.7`

Reply:
Ok I think I made the change requested